Dockerfile.coreSdk

```diff
@@ -46,5 +46,5 @@ ENV PATH=${PATH}:/usr/src/firebolt/mock-firebolt/cli
 WORKDIR /usr/src/firebolt/mock-firebolt/server
 
```

Comment:
Having both ".coreSdk" and ".allSdks" Dockerfiles is probably unnecessary now that things are merged together and you can't really have one SDK without the other. I think we should merge these / just have a "Dockerfile".

Reply:
Addressed